x-pack/plugins/files/common/types.ts

```diff
@@ -132,28 +132,39 @@ export type UpdatableFileAttributes<Meta = unknown> = Pick<FileJSON<Meta>, 'meta
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export type FileShareSavedObjectAttributes = {
-  created_at: string;
+  created: string;
+
+  /**
+   * Secret token used to access the associated file.
+   */
+  token: string;
```

Comment:
Added `token` here so that we can reference the file share instance without knowing the `token` value. Before, the `token` was the ID of the file share. This allows for more flexibility to not expose `token` values when we do not want to.

Current token values are only sent back when the share is first created.